x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformTask.java

```diff
@@ -237,6 +240,10 @@ public synchronized void start(Long startingCheckpoint, ActionListener<Response>
                 getTransformId()));
             return;
         }
+        if (shouldStopAtCheckpoint) {
```

Comment:
We don't want `start` to be called if we are `shouldStopAtCheckpoint`. 

Honestly, calling `start` here is pretty harmless, but I think it is better to throw to alert the user that "hey, we are stopping this thing soon".

Reply:
yes @davidkyle , as I said, calling `start` here should not cause problems, but if the user is attempting to start it after somebody already called `_stop?wait_for_checkpoint=true`, I think we should throw an error indicating that we are stopping